oaipmhTemplates/dha_RepoObject.xml

```diff
@@ -0,0 +1,154 @@
+                        </dha:hasIdentifier>
+                    </dha:Artefact>
+
+                    <dha:Organization rdf:parseType="Resource"> <!-- Q: How do distinguish between hasActor that points to an Organization rather than a Person? Use If conditions? -->
+                        <dha:hasCatchPhrase count="*" lang="true" val="@acdh:hasActor/hasTitle"></dha:hasCatchPhrase>
+                        <dha:hasIdentifier rdf:parseType="Resource">
+                            <premis:hasIdentifier count="1" val="@acdh:hasActor/hasIdentifier"></premis:hasIdentifier>
+                            <dha:conformsTo>https://id.acdh.oeaw.ac.at/</dha:conformsTo>
+                            <dha:hasParsableMetadata count="*" val="METAURL"></dha:hasParsableMetadata>
+                        </dha:hasIdentifier>
+                    </dha:Organization>
+
+                    <dha:Person rdf:parseType="Resource">
+                        <dha:hasCatchPhrase count="*" val="@acdh:hasActor/hasTitle"></dha:hasCatchPhrase>
+                        <dha:hasIdentifier rdf:parseType="Resource">
+                            <premis:hasIdentifier count="1" val="@acdh:hasActor/hasIdentifier"></premis:hasIdentifier>
+                            <dha:conformsTo>https://id.acdh.oeaw.ac.at/</dha:conformsTo>
+                            <dha:hasParsableMetadata count="*" val="METAURL"></dha:hasParsableMetadata>
+                        </dha:hasIdentifier>
+                    </dha:Person>
+
+                    <dha:Place rdf:parseType="Resource">
+                        <dha:hasCatchPhrase count="*" lang="true" val="@acdh:hasSpatialCoverage/hasTitle"></dha:hasCatchPhrase>
+                        <dha:hasIdentifier rdf:parseType="Resource">
+                            <premis:hasIdentifier count="*" val="@acdh:hasSpatialCoverage/hasIdentifier"></premis:hasIdentifier>
+                            <dha:conformsTo>https://id.acdh.oeaw.ac.at/</dha:conformsTo>
+                            <dha:hasParsableMetadata count="*" val="METAURL"></dha:hasParsableMetadata>
+                        </dha:hasIdentifier>
+                    </dha:Place>
+
+                </dha:NamedEntity>
+
+                <dha:Timespan rdf:parseType="Resource">
+                    <dha:hasCatchPhrase count="*" lang="true" val="/acdh:hasTemporalCoverage"></dha:hasCatchPhrase>
+                    <dha:hasIdentifier rdf:parseType="Resource">
+                        <premis:hasIdentifier count="*" val="/acdh:hasTemporalCoverageIdentifier"></premis:hasIdentifier>
+                        <dha:conformsTo></dha:conformsTo> <!-- It may be from perio.do or chronontology.dai or other sources, so it cannot be stated a priori -->
+                        <dha:hasParsableMetadata></dha:hasParsableMetadata>
+                    </dha:hasIdentifier>
+                    <dha:hasISODate></dha:hasISODate> <!-- Q: How to concatenate start and end coverage dates, using slash for duration? -->
```

Comment:
```
<dha:hasISODate>
  <foo val="/acdh:hasSomethingStart" replaceXMLTag="true"/> / 
  <foo val="/acdh:hasSomethingEnd" replaceXMLTag="true"/>
</dha:hasISODate>
```

Remarks:
* as the XML tag is replaced it doesn't matter what is tag's name (so I used foo)
* of course the separator between tags can be different or skipped at all